#request type
uint8 RESET = 0
uint8 START = 1
uint8 PAUSE = 2
uint8 RESUME = 3

uint8 request
string destination
